examples/jsm/loaders/VRMLLoader.js

```diff
@@ -830,7 +830,7 @@ var VRMLLoader = ( function () {
 					}
 
 				}
-
+				object.name=node.DEF;
```

Comment:
Please note that `DEF` is optional so it can be `undefined` and thus `Object3D.name`. This is not optional.

I suggest you change the lines to:
```js
if ( node.DEF !== undefined ) object.name = node.DEF;
```

Reply:
Yes. 
Is there anything else I need to do here - I'm not that familiar with github?

Reply:
The idea is that you push an additional commit with the requested changes.

Reply:
Maybe he did the PR from the website?

Reply:
@polygontwist If okay for you, I can do this change tomorrow. Otherwise I suggest you fork the repository, applying your changes in a new branch and then make a PR. So the typical contribution workflow.

Reply:
yes, please make it so.